engine/link/link.asm

```diff
@@ -2311,6 +2311,14 @@ WaitForLinkedFriend:
 	ldh [rSC], a
 	ld a, (1 << rSC_ON) | (0 << rSC_CLOCK)
 	vc_hook linkCable_fake_begin
```

Comment:
Add a comment above this `vc_hook` like the Celebi one, explaining briefly how it uses these values.